yang/vyatta-protocols-frr-isis-v1.yang

```diff
@@ -20,7 +20,9 @@ module vyatta-protocols-frr-isis-v1 {
   import vyatta-interfaces-loopback-v1 {
     prefix interfaces-loopback;
   }
-
+  import vyatta-interfaces-switch-v1 {
+    prefix interfaces-switch;
+  }
```

Comment:
Please maintain the blank line between here and the `organization` statement.

Reply:
Resolved.